pom.xml

```diff
@@ -44,8 +44,8 @@
 		<tycho.scmUrl>scm:git:https://github.com/eclipse-cdt/cdt</tycho.scmUrl>
 		<!-- Some old tests, like CDescriptorOldTests, fail due to reflection access. Therefore we add-opens to make that pass -->
 		<base.test.vmargs>-Xms256m -Xmx512m -ea --add-opens=java.xml/com.sun.org.apache.xerces.internal.dom=ALL-UNNAMED</base.test.vmargs>
-		<comparator.repo>https://download.eclipse.org/tools/cdt/releases/latest/</comparator.repo>
-		<api-baseline.repo>https://download.eclipse.org/tools/cdt/releases/latest/</api-baseline.repo>
+		<comparator.repo>https://ci.eclipse.org/cdt/job/cdt-jonahgraham-fork/job/fix-bree-11-3/5/artifact/releng/org.eclipse.cdt.repo/target/repository/</comparator.repo>
+		<api-baseline.repo>https://ci.eclipse.org/cdt/job/cdt-jonahgraham-fork/job/fix-bree-11-3/5/artifact/releng/org.eclipse.cdt.repo/target/repository/</api-baseline.repo>
```

Comment:
Once I publish an 11.3.1 build this should become the location of it before merging. For now just testing against my fork's build of 11.3.1

Reply:
Done.